pkg/bundle/bundle.go

```diff
@@ -35,51 +34,28 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/cert-manager/trust-manager/cmd/trust-manager/app/options"
```

Comment:
suggestion: I don't love the idea of depending on `./cmd` from `./pkg`.  I think that'll end in a circular dependency down the road - I'd expect `cmd` to depend on `pkg` but not the other way round.

I would vote to keep the options struct where it is in `pkg`.

(I totally accept that package structuring is a bit arbitrary but I do think it's better for the long term to not risk this kind of circular dependency creeping in)

Reply:
It was actually a circular dependency that made me move `Options`, but I agree with your comment. Will fix it.

Reply:
Would you be ok with moving the options package to `./pkg`? I cannot leave the Bundle options where it currently is located under `./pkg` - as that will create a circular dependency.